css/basscss.css

```diff
@@ -612,417 +706,19 @@ h6, .h6 { font-size: .75rem }
 
 .flex-justify  { -webkit-box-pack: justify; -webkit-justify-content: space-between; -ms-flex-pack: justify; justify-content: space-between }
 
-.flex-first { -webkit-box-ordinal-group: 0; -webkit-order: -1; -ms-flex-order: -1; order: -1 }
```

Comment:
I've found this to be really useful when doing responsive stuff layouts. The vendor prefixes make flex stuff like this a real hassle to use with inline styles.